book/website/ethical-research/ethics-open-source-governance.md

```diff
@@ -0,0 +1,99 @@
+(er-ethics-open-source-governance)=
+# Ethical Considerations when Choosing an Open Source Governance Model
+
+(er-ethics-open-source-governance-summary)=
+## Summary
+
+Open source projects are rooted in ethical principles. 
+Sharing the code so that anyone can use it, welcoming contributors and creating communities around specific projects are the core of open source. 
+But open source communities also have the responsibility to ensure they are a safe, inclusive environment, understand who might be affected by their projects or check that their tools are not being misused. 
+Therefore, Open Source Governance Models should embed those ethical considerations within their structure, but the framework to do so is far from well-established. 
+This chapter discusses some of the key points to take into consideration when designing and implementing ethical guidelines in governance models.
+
+
+(er-ethics-open-source-governance-motivation)=
+## Motivation and Background
+
+When setting up open source projects, it is important to pick a structure for organising the community that takes into account ethical considerations 
+The way that your governance model is designed has a significant impact on the ethics, diversity and inclusion in your project, so critically examining what the implied ethics are of a particular structure is a crucial part of the process of selecting one. 
+It can be difficult to know where to start when considering this aspect of open source projects, so this chapter offers some starting points to consider. 
```

Comment:
```suggestion
The way that your governance model is designed has a significant impact on the ethics, diversity and inclusion in your project, so critically examining what the implied ethics are of a particular structure is a crucial part of the process of selecting one. 
However, it is difficult to know where to start when planning governance structure for your open source projects.
It can be even more challenging when considering if a particular governance model align with ethical framework of your project.
Therefore, we have designed this chapter to offer some starting points to consider. 
```